impl.go

```diff
@@ -40,8 +40,26 @@ func findInterface(iface string) (path string, id string, err error) {
 		return "", "", fmt.Errorf("couldn't parse interface: %s", iface)
 	}
 
+	var importPath string
+	p := strings.Split(iface, "/")
```

Comment:
Since all we care about is the final /, if present, I think we can do this more cheaply and simply with strings.LastIndex and some string slicing.